src/MSBuild.UnitTests/XMake_Tests.cs

```diff
@@ -2042,6 +2032,33 @@ public void MultipleTargetsDoesNotCrash()
             logContents.ShouldContain("E2C73B5843F94B63B067D9BEB2C4EC52", () => logContents);
         }
 
+        [Theory]
+        [InlineData("-logger:,\"nonExistentlogger.dll\",IsOptional;Foo")]
+        [InlineData("-logger:ClassA,\"nonExistentlogger.dll\",IsOptional;Foo")]
+        [InlineData("-logger:,\"nonExistentlogger.dll\",IsOptional,OptionB,OptionC")]
+        [InlineData("-distributedlogger:,\"nonExistentlogger.dll\",IsOptional;Foo")]
+        [InlineData("-distributedlogger:ClassA,\"nonExistentlogger.dll\",IsOptional;Foo")]
+        [InlineData("-distributedlogger:,\"nonExistentlogger.dll\",IsOptional,OptionB,OptionC")]
```

Comment:
So the initial comma is required when specifying `IsOptional`, even if you're just passing an assembly? That seems ok, but worth calling out . . . somewhere. Not the CLI help since we're keeping that short.